Bearded.UI/EventArgs/ModifierKeys.cs

```diff
@@ -0,0 +1,30 @@
+using Bearded.Utilities.Input;
+using OpenTK.Input;
+
+namespace Bearded.UI.EventArgs
+{
+    public sealed class ModifierKeys
+    {
+        public bool Shift { get; }
+        public bool Control { get; }
+        public bool Alt { get; }
+        public bool Win { get; }
+
+        private ModifierKeys(bool shift, bool control, bool alt, bool win)
+        {
+            Shift = shift;
+            Control = control;
+            Alt = alt;
+            Win = win;
+        }
+
+        public static ModifierKeys FromInputManager(InputManager inputManager)
+        {
+            return new ModifierKeys(
+                inputManager.IsKeyPressed(Key.LShift) || inputManager.IsKeyPressed(Key.RShift),
+                inputManager.IsKeyPressed(Key.LControl) || inputManager.IsKeyPressed(Key.RControl),
+                inputManager.IsKeyPressed(Key.LAlt) || inputManager.IsKeyPressed(Key.RAlt),
+                inputManager.IsKeyPressed(Key.LWin) || inputManager.IsKeyPressed(Key.RWin));
```

Comment:
What's the argument for not exposing both separately as well? Is it that they are virtually never used differently? If so, is that a correct assumption?

Reply:
Yeah that's the assumption. We could always expose them as LShift and RShift as well, and have a Shift getter than is just the or of both, but it seems unlikely that it's needed, so it could be a YAGNI (you ain't gonna need it) feature. It can also be added in a backwards compatible way, so I'm happy to keep it as-is, but again, happy to change this is there's a good argument for it.

Reply:
Go ahead and merge.